R/quantity.R

```diff
@@ -1,9 +1,14 @@
+withDimensionExtensions <- "OSPSuite.Core.Domain.WithDimensionExtensions"

```

Comment:
@PavelBal Is this ok to define a constant like that? Just for this file. I was fedup of repeating it

Reply:
Don't see a problem with it. But maybe put it in UPPER_CASE according to our coding standards @msevestre